manage.py

```diff
@@ -145,7 +145,12 @@ def get_index_data_from_database(resource="all"):
             resource_table_=None, re_create_index=True, only_values=False
         )
         # validat ethe indexing
-        test_indexing_search_query(deep_check=True, check_studies=True)
+        try:
+            test_indexing_search_query(deep_check=False, check_studies=True)
+        except Exception:
+            pass

```

Comment:
What does this mean? e.g. a node is not reachable?

Reply:
Sorry, I used this `try except` block for testing on my local machine. I will remove it.